.credo.exs

```diff
@@ -0,0 +1,125 @@
+        {Credo.Check.Refactor.NegatedConditionsWithElse, []},
+        {Credo.Check.Refactor.Nesting, []},
+        {Credo.Check.Refactor.UnlessWithElse, []},
+        {Credo.Check.Refactor.WithClauses, []},
+
+        # Warnings
+        {Credo.Check.Warning.BoolOperationOnSameValues, []},
+        {Credo.Check.Warning.ExpensiveEmptyEnumCheck, []},
+        {Credo.Check.Warning.IExPry, []},
+        {Credo.Check.Warning.IoInspect, []},
+        {Credo.Check.Warning.LazyLogging, []},
+        {Credo.Check.Warning.MixEnv, false},
+        {Credo.Check.Warning.OperationOnSameValues, []},
+        {Credo.Check.Warning.OperationWithConstantResult, []},
+        {Credo.Check.Warning.RaiseInsideRescue, []},
+        {Credo.Check.Warning.UnusedEnumOperation, []},
+        {Credo.Check.Warning.UnusedFileOperation, []},
+        {Credo.Check.Warning.UnusedKeywordOperation, []},
+        {Credo.Check.Warning.UnusedListOperation, []},
+        {Credo.Check.Warning.UnusedPathOperation, []},
+        {Credo.Check.Warning.UnusedRegexOperation, []},
+        {Credo.Check.Warning.UnusedStringOperation, []},
+        {Credo.Check.Warning.UnusedTupleOperation, []},
+        {Credo.Check.Warning.UnsafeExec, []},
+
+        # Controversial and experimental checks (opt-in, just replace `false` with `[]`)
+        {Credo.Check.Readability.StrictModuleLayout, false},
+        {Credo.Check.Consistency.MultiAliasImportRequireUse, false},
+        {Credo.Check.Consistency.UnusedVariableNames, false},
+        {Credo.Check.Design.DuplicatedCode, false},
+        {Credo.Check.Readability.AliasAs, false},
+        {Credo.Check.Readability.MultiAlias, false},
+        {Credo.Check.Readability.Specs, false},
+        {Credo.Check.Readability.SinglePipe, false},
+        {Credo.Check.Readability.WithCustomTaggedTuple, false},
+        {Credo.Check.Refactor.ABCSize, false},
+        {Credo.Check.Refactor.AppendSingleItem, false},
+        {Credo.Check.Refactor.DoubleBooleanNegation, false},
+        {Credo.Check.Refactor.ModuleDependencies, false},
+        {Credo.Check.Refactor.NegatedIsNil, false},
+        {Credo.Check.Refactor.PipeChainStart, false},
+        {Credo.Check.Refactor.VariableRebinding, false},
+        {Credo.Check.Warning.LeakyEnvironment, false},
+        {Credo.Check.Warning.MapGetUnsafePass, false},
+        {Credo.Check.Warning.UnsafeToAtom, false},
+
+        # Nicene extras
+        {Nicene.AvoidForbiddenNamespaces, false},
+        # Following might be helpful, but is invisible to clients:
+        # consider in future
+        {Nicene.AvoidImportsFromCurrentApplication, false},
+        {Nicene.ConsistentFunctionDefinitions, false},
+        {Nicene.DocumentGraphqlSchema, false},
+        {Nicene.EctoSchemaDirectories, false},
+        {Nicene.FileAndModuleName, []},
+        {Nicene.FileTopToBottom, []},
+        {Nicene.NoSpecsPrivateFunctions, false},
+        {Nicene.PublicFunctionsFirst, []},
```

Comment:
What is this for?

Reply:
From the docs:
```
defmodule Nicene.FileAndModuleName do
  @moduledoc """
  Check to ensure that module names correspond to the file that it is defined it.
...
defmodule Nicene.FileTopToBottom do
  @moduledoc """
  Invokations of functions should be before definitions of those functions.
...
defmodule Nicene.NoSpecsPrivateFunctions do
  @moduledoc """
  Private functions should not have typespecs
...
defmodule Nicene.PublicFunctionsFirst do
  @moduledoc """
  Public functions in a module should be defined before private functions.
```

Reply:
Nevermind, let's go for it.

Reply:
The idea is to try these additions out, and drop them if they only block development. 
Now I'm not sure if this project is alive at all, given there're quite a lot of PRs now and it has quite strange bugs here and there (e.g. `PublicFunctionsFirst` doesn't take nested modules or module defs inside __using__ macro, and therefore gives false positives)